recipes/pyqtree/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pyqtree" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Pyqtree-{{ version }}.tar.gz
+  sha256: 4f36d5160ddf170d7245e9c7102a45211b85003383dd552b6cd109e50cc3af81
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - pyqtree
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://github.com/karimbahgat/Pyqtree
+  summary: A pure Python quad tree spatial index for GIS or rendering usage.
+  doc_url: https://pythonhosted.org/Pyqtree/
+  license: MIT
+  license_file: LICENSE
```

Comment:
Add comment that license file is manually packaged:

```
# License file manually packaged. See https://github.com/karimbahgat/Pyqtree/pull/20
license_file: LICENSE
...
```

This will be a reminder to check future releases to see if the license file has been included in the source distribution, at which point you can delete the manually included copy. Please note though that upstream currently, the license file is `license.txt`

Reply:
Thank you Joshua